online.codingblocks.com/components/AboutCourse/CourseRatingStats.vue

```diff
@@ -116,6 +116,12 @@ export default {
           stars: +key + 1
         }))
     }
+  },
+  filters: {
+    ensureAvatar (img) {
+      const random  = Math.floor(Math.random()*36) + 1
+      return img || `https://minio.codingblocks.com/img/avatar-${random}.svg`
```

Comment:
Put this logic in `utils`  dir and use from there. This will used in many places, so  it only makes sense we refactor it